build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
     `build-scan`
     `maven-publish`
     kotlin("jvm") version "1.3.31"
-    id("kotlinx-serialization") version "1.3.31"
+    id ("kotlinx-serialization") version "1.3.40"
```

Comment:
Let's use the latest version :) 1.3.41

Reply:
I will check I think there was an issue here during serialization of xml but I will check again